packages/istanbul-lib-source-maps/test/map-store.test.js

```diff
@@ -36,7 +36,7 @@ describe('map store', function () {
         },
         "end": {
           "line": 4,
-          "column": 5
+          "column": Infinity
```

Comment:
@loganfsmyth is it weird that line 4 (in a 5 line file) extends to the column position `Infinity`? Your comment indicates that this edge-case should only occur when we've hit the end of the file.

Reply:
The difficulty is that sourcemap mappings don't actually contain the end position data, so all you know is that it extends to the end of a given line, with no sense of what column that might be. You'd have to fetch the source content and split it by line to know the actual column offset.

I agree though that it may well require some downstream validation. I guess you could always just do `Number.MAX_SAFE_INTEGER` if you wanted to avoid dangers with possible `Infinity` values or something, but hard to say if that's actually any less likely to have issues.

Reply:
I haven't done a full review yet (I have other commitments today), but I just wanted to comment that I did check `Infinity` against the second argument to `String.slice`, `String.substr` and `String.substring`.  All worked as expected.

I'm could be convinced if someone could show a way that `Infinity` could break something but I'd prefer we did not use `Number.MAX_SAFE_INTEGER`.  `Infinity` is clearly not a specific number where `Number.MAX_SAFE_INTEGER` is.

Reply:
the fact that `Infinity` behaves nicely with the String helpers is good to know. We're talking about making this a breaking change any ways, and I'm feeling more and more confident about swapping out what were already undefined ranges with `Infinity1.

Reply:
Granted `undefined` also works with String helpers but does not work with comparisons.
```js
assert(5 < Infinity); // pass
assert(5 < undefined); // fail
```

So I think this is 1) an improvement and 2) appropriate to flag as a breaking change just in case.